client/src/dashboard/QuerySuggestionRewritingPath.js

```diff
@@ -15,6 +15,7 @@ import Title from './Title';
 import defaultSuggestionRewritingPathData from '../mock-api/suggestionRewritingPath';
 import SyntaxHighlighter from 'react-syntax-highlighter';
 import { vs } from 'react-syntax-highlighter/dist/esm/styles/hljs';
+import {userContext} from "../userContext";
```

Comment:
Let's include all changes to this file to the next PR which is about `Add to Mine` button.